### ISSUE TITLE
Feature Request:  Event Hub - Auto Inflate

### ISSUE THREAD

**Issue by WeAreGroot** [terraform-providers/terraform-provider-azurerm#168]
Microsoft released a new EventHub feature called AutoInflate to scale up the number of throughput inputs (TUs). 

https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-auto-inflate

The following property needs to be added to the ARM template to enable this feature in the Event Hub namespace:
```
"properties": {
  "isAutoInflateEnabled": true,
  "maximumThroughputUnits": 10
},
```

It'd be great to get this onto the Terraform resource.

### Affected Resource(s)
Please list the resources as a list, for example:
- azurerm_eventhub

Thanks all, 

-rick-

**Comment by tombuildsstuff** [comment]
Hey @WeAreGroot 

Thanks for opening this issue

I've taken a quick look into this and we're missing a couple of properties needed in the Azure SDK to make this happen - however [I've opened this issue](https://github.com/Azure/azure-sdk-for-go/issues/721) to request the SDK is updated to the latest version of the API - and once that's done we can investigate adding support for these, which looks like it should be fairly quick :)

Thanks!

**Comment by katbyte** [comment]
Hey @WeAreGroot,

It seems this was added last November and released as part of `1.0.0` so I'm going to close this ticket. Seems like it slipped through the cracks 😅
